kili/mutations/issue/__init__.py

```diff
@@ -0,0 +1,85 @@
+"""
+Issue mutations
+"""
+
+from typing import Literal, Optional
+from typeguard import typechecked
+from .helpers import get_issue_number
+from ...helpers import Compatible, format_result
+from .queries import GQL_APPEND_TO_ISSUES
+from ..comment.queries import GQL_APPEND_TO_COMMENTS
+from ...queries.label import QueriesLabel
+
+
+class MutationsIssue:
+    """Set of Issue mutations."""
+
+    # pylint: disable=too-few-public-methods,too-many-arguments,too-many-locals
+
+    def __init__(self, auth):
+        """Initialize the subclass.
+
+        Args:
+            auth: KiliAuth object
+        """
+        self.auth = auth
+
+    @Compatible(["v1", "v2"])
+    @typechecked
+    def append_to_issues(
+        self,
+        label_id: str,
+        project_id: str,
+        object_mid: Optional[str] = None,
+        text: Optional[str] = None,
+        type_: Literal["ISSUE", "QUESTION"] = "ISSUE",
+    ) -> dict:
+        """Create an issue.
+
+        Args:
+            label_id: Id of the label to add an issue to
+            object_mid: Mid of the object in the label to associate the issue to
+            type_: type of the issue to add. Can be either "ISSUE" or "QUESTION"
+            text: If given, write a comment related to the issue
+            project_id: Id of the project
+
+        Returns:
+            A result object which indicates if the mutation was successful,
+                or an error message.
+        """
+        issue_number = get_issue_number(self.auth, project_id, type_)
+        try:
+            asset_id = QueriesLabel(self.auth).labels(
+                project_id=project_id, label_id=label_id, fields=["labelOf.id"], disable_tqdm=True
+            )[0]["labelOf"]["id"]
+        except:
+            # pylint: disable=raise-missing-from
+            raise ValueError(
+                f"Label ID {label_id} does not exist in the project of ID {project_id}"
+            )
+        variables = {
+            "data": {
+                "issueNumber": issue_number,
+                "labelID": label_id,
+                "objectMid": object_mid,
+                "type": type_,
+            },
+            "where": {"id": asset_id},
+        }
+        result = self.auth.client.execute(GQL_APPEND_TO_ISSUES, variables)
+        formated_result = format_result("data", result)
+        issue_id = formated_result["id"]
+
+        if text:
+            variables = {
+                "data": {
+                    "text": text,
+                    "inReview": False,
```

Comment:
`inReview` should be a parameter to the method? If deprecated but mandatory to pass, you can add a comment to indicate that

Reply:
This is quite tricky, inReview is a mandatory parameter but only useful when calling the append_to_comments mutation from the front. It does not have any legitimity on the SDK. 
In the front, it determines if the comment is posted in the review panel by a person with review rights. If yes, and if the comment has a question type, then it tags the comment as the answer of a question. 
Here, as the comment is the first comment to be posted after the issue/question creation, it is not an answer and there is no review process in the SDK so I always keep it to false. 
If one day we want to be able to add answers to comments though the SDK, we will have to think about that. ( Edouard first wrote a append_to_comments function in the SDK but I deleted it because Damien didn't need it and because this case makes the function ambigious in some cases)

Reply:
```suggestion
                    "inReview": False,  # mandatory and always False (can be True when used in the frontend
```